src/olympia/files/utils.py

```diff
@@ -652,6 +655,7 @@ class FSyncMixin(object):
     We need this to make sure that on EFS / NFS all data is immediately
     written to avoid any data loss on the way.
     """
+
```

Comment:
🐼

Reply:
That's my auto-formatter doing the wrong thing. Let me get rid of those.